### PR TITLE
feat: support tailwind and daisyUI

### PR DESCRIPTION
**Pull request by huybery**
As discussed in #217 , supported tailwind and daisyUI and made some styling improvements, not perfect but we can keep iterating.

image

### CODE REVIEW

**Review by yimothysu — APPROVED** [review]

File: frontend/src/App.css [hunk 1]
```diff
@@ -1,4 +1,8 @@
 /* App.css */
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
 .app {
   display: flex;
```
Comment (rbren) [inline frontend/src/App.css:7]: should be able to remove this line now that you have the `flex` class.

In general, the hope with tailwind is that we can remove _all_ of these CSS styles, or at least 95% of them. Can probably do that in future PRs though

File: frontend/src/components/CodeEditor.tsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ function CodeEditor(): JSX.Element {
       }}
     >
       <Editor
-        height="95%"
+        height="100%"
```
Comment (rbren) [inline frontend/src/components/CodeEditor.tsx:19]: you've got class `h-full` now

**Comment by rbren** [comment]
Looks good overall! Should we just delete the commented code?

**Comment by huybery** [comment]
> Looks good overall! Should we just delete the commented code?

The browser part is not known to be implemented at the moment, so I chose to comment it out.

**Comment by rbren** [comment]
Fair enough! I typically suggest just deleting code instead of commenting--we can always get back at it through the git history. But either way LGTM!

**Pull request merged**